metasyn/demo/dataset.py

```diff
@@ -135,6 +139,39 @@ def name(self):
     def schema(self):
         return {"SOP_DESCRIPTION": pl.Categorical, "BODYSITE_DESCRIPTION": pl.Categorical}
 
+
+@register
+class HospitalDataset(BaseDataset):
+    """Example electronic health record hospital dataset. This dataset was created manually."""
+
+    @property
+    def name(self):
+        return "hospital"
+
+    @property
+    def schema(self):
+        return {"date_admitted": pl.Date, "time_admitted": pl.Time, "type": pl.Categorical}
+
+
+@register
+class DrugUseDataset(BaseDataset):
+    """Example dataset with answers to an open question on study participants' daily drug use.
+
+    This example dataset was generated through ChatGPT-4o on 07-11-2024 using the following prompt:
```

Comment:
I kind of wonder if this dataset in itself might be a privacy risk? What if ChatGPT was trained on real data? Probably we should not worry about this..?